llvm/test/DebugInfo/Symbolize/XCOFF/xcoff-symbolize-data.ll

```diff
@@ -0,0 +1,149 @@
+; RUN: llc -filetype=obj -o %t -mtriple=powerpc-aix-ibm-xcoff < %s
+; RUN: llvm-symbolizer --obj=%t 'DATA 0x60' 'DATA 0x61' 'DATA 0x64' 'DATA 0X68' \
+; RUN:   'DATA 0x90' 'DATA 0x94' 'DATA 0X98' | FileCheck %s
+
+; CHECK: bss_global
+; CHECK-NEXT: 96 4
+; CHECK-NEXT: /t.cpp:1
+; CHECK-EMPTY:
+
+; CHECK: bss_global
+; CHECK-NEXT: 96 4
+; CHECK-NEXT: /t.cpp:1
+; CHECK-EMPTY:
+
+; CHECK: data_global
+; CHECK-NEXT: 100 4
+; CHECK-NEXT: /t.cpp:2
+; CHECK-EMPTY:
+
+; CHECK: str
+; CHECK-NEXT: 104 4
+; CHECK-NEXT: /t.cpp:4
+; CHECK-EMPTY:
+
+; FIXME: fix the wrong size 152
+; CHECK: f()::function_global
+; CHECK-NEXT: 144 152
+; CHECK-NEXT: /t.cpp:8
+; CHECK-EMPTY:
+
+; FIXME: fix the wrong size 152
```

Comment:
I have a preference for comments to use double comment markers in lit tests, to help them stand out from CHECK and RUN lines, so `;;` in this context.